pages/admin/index.tsx

```diff
@@ -37,6 +37,10 @@ export default function Admin({ questions }: { questions: QADocument[] }) {
   };
 
   const postAnnouncement = async () => {
+    if (!user.permissions.includes('super_admin')) {
+      alert('You do not have permission to perform this functionality');
+      return;
+    }
```

Comment:
@abdullah50053 Shouldn't admins be allowed to make announcements? For example, shouldn't the person on-call at the front desk be able to do this?

Reply:
It's up to the organizers at the end of the day, but admins usually have their role assigned before the hackathon and keep it after the hackathon, and no one deletes FCM tokens. Mostly a safety measure to prevent admins from accidentally sending something out